cmd/start.go

```diff
@@ -45,7 +48,10 @@ const startDescription = `Starts the Keep client in the foreground. Currently th
 // Values related with balance monitoring.
 // defaultBalanceAlertThreshold determines the alert threshold below which
 // the alert should be triggered.
-var defaultBalanceAlertThreshold = big.NewInt(500000000000000000) // 0.5 ether
+var defaultBalanceAlertThreshold = commoneth.WrapWei(
+	big.NewInt(500000000000000000),
+)
```

Comment:
Non-blocking and not necessarily in this PR but I think it's worth considering to have `ethereum.Wei` function in`keep-core` that would call `keep-common/pkg/chain/ethereum` internally. 

This is an additional layer but it feels good when the only package talking with `keep-common`'s ethereum utils is `ethereum`.

Reply:
We can but I don't want to go too far in this PR. I Will post a TODO: https://github.com/keep-network/keep-core/commit/3f9f104b3d8be0b44ee6adb230d60314551487cc